pkgs/development/tools/language-servers/emmet-language-server/default.nix

```
@@ -0,0 +1,33 @@
+{ lib
+, buildNpmPackage
+, fetchFromGitHub
+, fetchpatch
+}:
+
+buildNpmPackage rec {
+  pname = "emmet-language-server";
+  version = "v2.2.0";
```

```suggestion
  version = "2.2.0";
```
Version should be just a number, no `v` prefix, see https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming